scene/gui/scroll_container.cpp

```diff
@@ -824,3 +909,29 @@ ScrollContainer::ScrollContainer() {
 
 	set_clip_contents(true);
 }
+
+void ScrollContainer::set_horizontal_content_align(int p_align) {
+	int clamped = CLAMP(p_align, 0, 2);
```

Comment:
This shouldn't use clamp, it should use a range check instead, also those should be in the correct order in the file, above where `set_h_scroll` is declared